solutions/account-infrastructure-base/main.tf

```diff
@@ -77,4 +238,21 @@ module "account_infrastructure_base" {
   provision_trusted_profile_projects = var.provision_trusted_profile_projects
   trusted_profile_description        = var.trusted_profile_description
   trusted_profile_roles              = var.trusted_profile_roles
+
+  # cbr
+  provision_cbr                              = var.provision_cbr
+  cbr_prefix                                 = var.prefix
```

Comment:
`cbr_prefix` value cannot be set to `null` by default as this is causing an issue when CBR FSCloud module is called at this [line](https://github.com/terraform-ibm-modules/terraform-ibm-cbr/blob/8229771d947994a93e1f2dbb3cbc19fe4026244d/modules/fscloud/main.tf#L456).
```
We may need to check if the `prefix` value is null and then assign some string to `cbr_prefix` or expose `cbr_prefix` variable for the user.
│   on .terraform/modules/account_infrastructure_base.cbr_fscloud/modules/fscloud/main.tf line 423, in module "cbr_rule":
│  423:   rule_description = try(each.value.description, null) != null ? each.value.description : "${var.prefix}-${each.key}-rule"
│     ├────────────────
│     │ var.prefix is null
│ 
│ The expression result is null. Cannot include a null value in a string
│ template.
```